de/css/dita-ot-doc.css

```diff
@@ -1,22 +1,48 @@
 /* DITA Open Toolkit documentation style */
 
+/****************************************
+Import Prism
+****************************************/
+
+@import url("https://prismjs.com/themes/prism.css");
+
 /****************************************
 Basic fonts and typography
 ****************************************/
 
+@font-face {
+  font-family: JuniusVF;
+  src: url('https://psb1558.github.io/Junicode-New/webfiles/JuniusVF.woff2');
+  font-weight: 300 700;
+  font-stretch: 60% 100%;
+  font-style: normal;
+}
+
+
+
 article {
   color: #333;
   position: relative;
 }
 
+
+
 body {
   background-color: white;
   background-image: -webkit-linear-gradient(left, #eee 0%, #fff 300px); /* Chrome10+,Safari5.1+ */
   background-image:    -moz-linear-gradient(left, #eee 0%, #fff 300px); /* FF3.6+ */
   background-image:         linear-gradient(left, #eee 0%, #fff 300px); /* W3C */
   color: #333;
+  font-family: Bitter, 'Noto Serif', Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
Shouldn't this include `JuniusVF`?

Reply:
No. JuniusVF is mainly important for the examples in the tables. JuniusVF is also a serif font, but the general text should be sans serif (Helvetica, Arial...).

Reply:
Understood, but since we already have it in the CSS, it might be a good idea to add `JuniusVF` at the end of the list as a fallback for codepoints not supported by the other fonts.